### ISSUE TITLE
MMAgent - updated product ID

### ISSUE THREAD

**Issue by janegilring** [rchaganti/DSCResources#11]
It seems like the MM Agent has gotten a new product ID. At least the one I[`m using ](https://opsinsight.blob.core.windows.net/publicfiles/MMASetup-AMD64.exe)got a different ID than the one in the DSC Resource:
8A7F2C51-4C7D-4BFD-9014-91D11F24AAE2

**Comment by janegilring** [comment]
Turns out there are multiple versions in play here:

Version: 7.2.10375.0
ID: E854571C-3C01-4128-99B8-52512F44E5E9

Version: 8.0.10879.0
ID: 8A7F2C51-4C7D-4BFD-9014-91D11F24AAE2

Version: Unknown (the one in cMMAgentInstall
ID: E854571C-3C01-4128-99B8-52512F44E5E9

**Comment by rchaganti** [comment]
I am pushing an update very soon! The existing composite resource will always have a problem when they change the product ID. I will create a separate resource for this.

**Comment by janegilring** [comment]
Maybe consider to add the product ID as a parameter (with a default value)?
Or accept a -Version parameter and have a mapping table between versions and product IDs which will make the resource automatically use the correct ID for the specified version?
Although it might be a challenge to maintain, as I`m not sure how often the product ID is changed.

**Comment by rchaganti** [comment]
Certainly. But, I am thinking of avoiding Package resource completely.
